app/script/conversation/ConversationRepository.js

```diff
@@ -1837,22 +1837,22 @@ z.conversation.ConversationRepository = class ConversationRepository {
    * @returns {Promise} Resolves when sent status was updated
    */
   _update_message_as_sent(conversation_et, message, event_time) {
```

Comment:
Renaming `message` into `message_et` here would also do the trick to make Travis happy.